src/Pages/Login/login.css

```diff
@@ -6,14 +6,14 @@
 }
 
 .txtb {
-  border-bottom: 2px solid #87b0c7; 
+  border-bottom: 2px solid #87b0c7;
```

Comment:
[Button](https://cdn.discordapp.com/attachments/555072625945214986/1117315865617104896/image.png)

Can we make the blue darker for a little more contrast? e.g. [#768ecd](https://cdn.discordapp.com/attachments/555072625945214986/1117316286347755560/image.png)